packages/editor/src/components/parts/program/activity/ProgramInterfaceStartPart.tsx

```diff
@@ -32,8 +32,8 @@ const ProgramInterfaceStartPart = ({ thirdParty }: { thirdParty?: boolean }) =>
 
   return (
     <>
-      {(thirdParty == undefined || thirdParty == false) && (
-        <JavaClassSelector javaClass={config.javaClass} onChange={change => update('javaClass', change)} type='ACTIVITY' />
+      {!thirdParty && (
+        <JavaClassSelector javaClass={config.javaClass} onChange={change => update('javaClass', change)} type='INTERMEDIATE' />

```

Comment:
are you sure about this type change?

Reply:
yes it works like that, I was also a bit confused but I checked it. 👍

Reply:
isn't this the type you will use the show already implemented beans in the project and to for the 'create' action?
ch.ivyteam.ivy.lsp.inscription.meta.program.impl.DefaultProgramMeta.types(ProgramInterfacesRequest)
... if so the type matters. If it doesn't match the 'element' nature, it will list and create invalid java impls.

Reply:
Oops, sorry, yes you are right. A copy past error. I have now changed it back to ACTIVITY 👍